CRITICAL - 2024-07-21 00:30:58 --> ParseError: syntax error, unexpected token "public"
[Method: POST, Route: enviarlogin]
in APPPATH\Controllers\login_controller.php on line 63.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ProT3_24798071\\app\\Controllers\\login_controller.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\login_controller')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\login_controller')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\login_controller', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 00:30:58 --> ParseError: syntax error, unexpected token "public"
[Method: POST, Route: enviarlogin]
in APPPATH\Controllers\login_controller.php on line 63.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ProT3_24798071\\app\\Controllers\\login_controller.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\login_controller')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\login_controller')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\login_controller', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 00:40:23 --> ParseError: syntax error, unexpected token "public"
[Method: POST, Route: enviarlogin]
in APPPATH\Controllers\login_controller.php on line 63.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ProT3_24798071\\app\\Controllers\\login_controller.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\login_controller')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\login_controller')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\login_controller', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 01:01:32 --> Error: Class "App\Controllers\Usuario_Model" not found
[Method: POST, Route: enviar-form]
in APPPATH\Controllers\usuario_controller.php on line 30.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 01:09:06 --> ErrorException: Undefined variable $id_usuario
[Method: POST, Route: enviar-form]
in APPPATH\Controllers\usuario_controller.php on line 26.
 1 APPPATH\Controllers\usuario_controller.php(26): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $id_usuario', 'C:\\xampp\\htdocs\\ProT3_24798071\\app\\Controllers\\usuario_controller.php', 26)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-21 01:10:33 --> mysqli_sql_exception: Table 'caballero_g.usuario' doesn't exist in C:\xampp\htdocs\ProT3_24798071\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\ProT3_24798071\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\ProT3_24798071\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\ProT3_24798071\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\ProT3_24798071\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ProT3_24798071\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ProT3_24798071\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('grisk_75@hotmai...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\ProT3_24798071\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'grisk_75@hotmai...', Array, Array, 'email')
#7 C:\xampp\htdocs\ProT3_24798071\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ProT3_24798071\app\Controllers\usuario_controller.php(23): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ProT3_24798071\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\ProT3_24798071\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\ProT3_24798071\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_24798071\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_24798071\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_24798071\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-07-21 01:10:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'caballero_g.usuario' doesn't exist
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('grisk_75@hotmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'grisk_75@hotmail.com', [...], [...], 'email')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\usuario_controller.php(23): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 01:10:33 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'caballero_g.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'grisk_75@hotmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'grisk_75@hotmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('grisk_75@hotmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'grisk_75@hotmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(23): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 01:10:33 --> [Caused by] mysqli_sql_exception: Table 'caballero_g.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `usuario`
WHERE `email` = \'grisk_75@hotmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'grisk_75@hotmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'grisk_75@hotmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('grisk_75@hotmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'grisk_75@hotmail.com', [...], [...], 'email')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\usuario_controller.php(23): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 05:58:16 --> Error: Class "App\Models\Model" not found
[Method: POST, Route: enviar-form]
in APPPATH\Models\Usuario_Model.php on line 5.
 1 SYSTEMPATH\Autoloader\Autoloader.php(317): include_once()
 2 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ProT3_24798071\\app\\Models\\Usuario_Model.php')
 3 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Models\\Usuario_Model')
 4 APPPATH\Controllers\usuario_controller.php(30): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Models\\Usuario_Model')
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 6 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 06:18:16 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: POST, Route: enviar-form]
in APPPATH\Controllers\usuario_controller.php on line 3.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-21 06:19:21 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: POST, Route: enviar-form]
in APPPATH\Controllers\usuario_controller.php on line 3.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-21 06:20:35 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: POST, Route: enviar-form]
in APPPATH\Controllers\usuario_controller.php on line 3.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-21 06:22:01 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: POST, Route: enviar-form]
in APPPATH\Controllers\usuario_controller.php on line 3.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-21 06:33:05 --> ParseError: syntax error, unexpected token "public"
[Method: POST, Route: enviarlogin]
in APPPATH\Controllers\login_controller.php on line 63.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ProT3_24798071\\app\\Controllers\\login_controller.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\login_controller')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\login_controller')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\login_controller', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 06:42:10 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: POST, Route: enviar-form]
in APPPATH\Controllers\usuario_controller.php on line 2.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-21 06:49:03 --> ErrorException: Namespace declaration statement has to be the very first statement or after any declare call in the script
[Method: POST, Route: enviar-form]
in APPPATH\Controllers\usuario_controller.php on line 2.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-07-21 06:52:51 --> mysqli_sql_exception: Unknown column 'usuario' in 'field list' in C:\xampp\htdocs\ProT3_24798071\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\ProT3_24798071\system\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\ProT3_24798071\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\ProT3_24798071\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\ProT3_24798071\system\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\ProT3_24798071\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\ProT3_24798071\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\ProT3_24798071\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\ProT3_24798071\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\ProT3_24798071\app\Controllers\usuario_controller.php(55): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\ProT3_24798071\system\CodeIgniter.php(933): App\Controllers\Usuario_Controller->formValidation()
#10 C:\xampp\htdocs\ProT3_24798071\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario_Controller))
#11 C:\xampp\htdocs\ProT3_24798071\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_24798071\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_24798071\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_24798071\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-07-21 06:52:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'usuario' in 'field list'
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `usuario`, `email`) VALUES (:nombre:, :apellido:, :usuario:, :email:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\usuario_controller.php(55): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuario_Controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario_Controller))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 06:52:51 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'usuario' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `usuario`, `email`) VALUES (\'GRISELDA ANTONIA\', \'CABALLERO\', \'gris\', \'grisk_75@hotmail.com\')')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `usuario`, `email`) VALUES (\'GRISELDA ANTONIA\', \'CABALLERO\', \'gris\', \'grisk_75@hotmail.com\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `usuario`, `email`) VALUES (:nombre:, :apellido:, :usuario:, :email:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\usuario_controller.php(55): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuario_Controller->formValidation()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario_Controller))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 06:52:51 --> [Caused by] mysqli_sql_exception: Unknown column 'usuario' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `usuario`, `email`) VALUES (\'GRISELDA ANTONIA\', \'CABALLERO\', \'gris\', \'grisk_75@hotmail.com\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `apellido`, `usuario`, `email`) VALUES (\'GRISELDA ANTONIA\', \'CABALLERO\', \'gris\', \'grisk_75@hotmail.com\')')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `apellido`, `usuario`, `email`) VALUES (\'GRISELDA ANTONIA\', \'CABALLERO\', \'gris\', \'grisk_75@hotmail.com\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2333): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `apellido`, `usuario`, `email`) VALUES (:nombre:, :apellido:, :usuario:, :email:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\usuario_controller.php(55): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuario_Controller->formValidation()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario_Controller))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 16:03:42 --> ParseError: syntax error, unexpected token "public"
[Method: POST, Route: enviarlogin]
in APPPATH\Controllers\login_controller.php on line 63.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ProT3_24798071\\app\\Controllers\\login_controller.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\login_controller')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\login_controller')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\login_controller', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 16:16:56 --> ParseError: syntax error, unexpected token "public"
[Method: POST, Route: enviarlogin]
in APPPATH\Controllers\login_controller.php on line 62.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ProT3_24798071\\app\\Controllers\\login_controller.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\login_controller')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\login_controller')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\login_controller', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-21 17:52:18 --> mysqli_sql_exception: Table 'caballero_g.usuario' doesn't exist in C:\xampp\htdocs\ProT3_24798071\system\Database\MySQLi\Connection.php:310
Stack trace:
#0 C:\xampp\htdocs\ProT3_24798071\system\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1\nFROM `...', 0)
#1 C:\xampp\htdocs\ProT3_24798071\system\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1\nFROM `...')
#2 C:\xampp\htdocs\ProT3_24798071\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1\nFROM `...')
#3 C:\xampp\htdocs\ProT3_24798071\system\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1\nFROM `...', Array, false)
#4 C:\xampp\htdocs\ProT3_24798071\system\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\ProT3_24798071\system\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('grisk_75@hotmai...', 'email', Array, NULL, 'email')
#6 C:\xampp\htdocs\ProT3_24798071\system\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'grisk_75@hotmai...', Array, Array, 'email')
#7 C:\xampp\htdocs\ProT3_24798071\system\Controller.php(137): CodeIgniter\Validation\Validation->run()
#8 C:\xampp\htdocs\ProT3_24798071\app\Controllers\usuario_controller.php(24): CodeIgniter\Controller->validate(Array)
#9 C:\xampp\htdocs\ProT3_24798071\system\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
#10 C:\xampp\htdocs\ProT3_24798071\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
#11 C:\xampp\htdocs\ProT3_24798071\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\ProT3_24798071\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\ProT3_24798071\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\ProT3_24798071\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-07-21 17:52:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'caballero_g.usuario' doesn't exist
[Method: POST, Route: enviar-form]
in SYSTEMPATH\Database\BaseConnection.php on line 686.
 1 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('grisk_75@hotmail.com', 'email', [...], null, 'email')
 4 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'grisk_75@hotmail.com', [...], [...], 'email')
 5 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 6 APPPATH\Controllers\usuario_controller.php(24): CodeIgniter\Controller->validate([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 17:52:18 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'caballero_g.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 315.
 1 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'grisk_75@hotmail.com\'
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'grisk_75@hotmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('grisk_75@hotmail.com', 'email', [...], null, 'email')
 6 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'grisk_75@hotmail.com', [...], [...], 'email')
 7 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 8 APPPATH\Controllers\usuario_controller.php(24): CodeIgniter\Controller->validate([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 17:52:18 --> [Caused by] mysqli_sql_exception: Table 'caballero_g.usuario' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 310.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(310): mysqli->query('SELECT 1
FROM `usuario`
WHERE `email` = \'grisk_75@hotmail.com\'
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(732): CodeIgniter\Database\MySQLi\Connection->execute('SELECT 1
FROM `usuario`
WHERE `email` = \'grisk_75@hotmail.com\'
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT 1
FROM `usuario`
WHERE `email` = \'grisk_75@hotmail.com\'
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1637): CodeIgniter\Database\BaseConnection->query('SELECT 1
FROM `usuario`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Validation\StrictRules\Rules.php(228): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\Validation\Validation.php(336): CodeIgniter\Validation\StrictRules\Rules->is_unique('grisk_75@hotmail.com', 'email', [...], null, 'email')
 7 SYSTEMPATH\Validation\Validation.php(201): CodeIgniter\Validation\Validation->processRules('email', 'email', 'grisk_75@hotmail.com', [...], [...], 'email')
 8 SYSTEMPATH\Controller.php(137): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\usuario_controller.php(24): CodeIgniter\Controller->validate([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 19:15:01 --> ArgumentCountError: Too few arguments to function CodeIgniter\HTTP\Response::redirect(), 0 passed in C:\xampp\htdocs\ProT3_24798071\app\Controllers\usuario_controller.php on line 51 and at least 1 expected
[Method: POST, Route: enviar-form]
in SYSTEMPATH\HTTP\ResponseTrait.php on line 445.
 1 APPPATH\Controllers\usuario_controller.php(51): CodeIgniter\HTTP\Response->redirect()
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\usuario_controller->formValidation()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\usuario_controller))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 19:42:33 --> Error: Interface "App\Filters\FiltersInterface" not found
[Method: GET, Route: panel]
in APPPATH\Filters\auth.php on line 7.
 1 SYSTEMPATH\Autoloader\Autoloader.php(317): include_once()
 2 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\ProT3_24798071\\app\\Filters\\Auth.php')
 3 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Filters\\Auth')
 4 SYSTEMPATH\Filters\Filters.php(197): CodeIgniter\Autoloader\Autoloader->loadClass('App\\Filters\\Auth')
 5 SYSTEMPATH\Filters\Filters.php(184): CodeIgniter\Filters\Filters->runBefore([...])
 6 SYSTEMPATH\CodeIgniter.php(481): CodeIgniter\Filters\Filters->run('panel', 'before')
 7 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-21 19:48:24 --> ErrorException: Could not check compatibility between App\Filters\Auth::before(App\Filters\RequesInterface $request, $arguments = null) and CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null), because class App\Filters\RequesInterface is not available
[Method: GET, Route: panel]
in APPPATH\Filters\auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-21 20:04:26 --> ErrorException: Could not check compatibility between App\Filters\Auth::before(App\Filters\RequesInterface $request, $arguments = null) and CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null), because class App\Filters\RequesInterface is not available
[Method: GET, Route: panel]
in APPPATH\Filters\auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-21 20:04:43 --> ErrorException: Could not check compatibility between App\Filters\Auth::before(App\Filters\RequesInterface $request, $arguments = null) and CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null), because class App\Filters\RequesInterface is not available
[Method: GET, Route: panel]
in APPPATH\Filters\auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-21 20:05:40 --> ErrorException: Could not check compatibility between App\Filters\Auth::before(App\Filters\RequesInterface $request, $arguments = null) and CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null), because class App\Filters\RequesInterface is not available
[Method: GET, Route: panel]
in APPPATH\Filters\auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-21 20:06:10 --> ErrorException: Could not check compatibility between App\Filters\Auth::before(App\Filters\RequesInterface $request, $arguments = null) and CodeIgniter\Filters\FilterInterface::before(CodeIgniter\HTTP\RequestInterface $request, $arguments = null), because class App\Filters\RequesInterface is not available
[Method: GET, Route: panel]
in APPPATH\Filters\auth.php on line 9.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
